src/selectors/balances.js

```diff
@@ -82,3 +89,10 @@ export const paymentNetworkTotalBalanceSelector: (RootReducerState) => BigNumber
   },
 );
 
+export const useWalletAssetBalance = (chain: ?Chain, assetAddress: ?string): BigNumber => {
+  const accountBalances = useRootSelector(accountAssetsBalancesSelector);
+  if (!chain || !assetAddress) return BigNumber(0);
+
+  const balance = valueForAddress(accountBalances[chain]?.wallet, assetAddress)?.balance;
+  return BigNumber(balance ?? 0);
```

Comment:
I think this can be combined with `getWalletBalanceForAsset`.

Reply:
Changed